jobs/build/merge_ocp/merge_ocp.sh

```diff
@@ -0,0 +1,30 @@
+# usage: ./merge_ocp.sh ./working/ enterprise-3.11 release-3.11
+set -o xtrace
```

Comment:
```suggestion
set -e
set -x
```

`-e` for exit on error and `-o` is just another way to print out commands before executing (like how you did `xtrace` before).

Reply:
Also probably enable the one for exiting on undefined variables.

Reply:
`set -euxo pipefail` is sort of like `gcc -Wall` . . .

Reply:
Ack. Will change.